lib/active_model_serializers/rspec/matchers/serializer.rb

```diff
@@ -0,0 +1,57 @@
+require 'active_model_serializers/test/serializer'
+
+module ActiveModelSerializers
+  module RSpecMatchers
+    module Serializer
+      extend ActiveModelSerializers::Test::Serializer
+      extend ActiveSupport::Concern
+
+      included do
+        setup :setup_serialization_subscriptions
+        teardown :teardown_serialization_subscriptions
+
+        RSpec::Matchers.define :use_serializer do |expected|
```

Comment:
I agree, for me the JSON Schema is more important, because no matter what serializer is being used I simply want the response to be what I expect. That said, I was simply translating the current asserts to it's RSpec equivalents.
 
These `setup` and `teardown` set the subscriber needed to know what Serializer is being used on the render.